nginx/nginx.conf

```diff
@@ -0,0 +1,71 @@
+worker_processes 1;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+    sendfile        on;
+    keepalive_timeout  65;
```

Comment:
double check that file uploading still works
check if need to increase max payload size and timeout